vertx-mssql-client/src/main/java/io/vertx/mssqlclient/MSSQLConnectOptions.java

```diff
@@ -130,6 +128,30 @@ public MSSQLConnectOptions addProperty(String key, String value) {
     return (MSSQLConnectOptions) super.addProperty(key, value);
   }
 
+  /**
+   * Get the desired size (in bytes) for TDS packets.
+   *
+   * @return the desired packet size
+   */
+  public int getPacketSize() {
+    return packetSize;
+  }
+
+  /**
+   * Set the desired size (in bytes) for TDS packets.
+   * <p>
+   * The client will use the value as a parameter in the LOGIN7 packet.
+   * The server may or may not accept it.
+   *
+   * @param packetSize the desired packet size (in bytes)
+   * @return a reference to this, so the API can be used fluently
+   * @throws IllegalArgumentException if {@code packetSize} is smaller than {@link #MIN_PACKET_SIZE} or bigger than {@link #MAX_PACKET_SIZE}
+   */
+  public MSSQLConnectOptions setPacketSize(int packetSize) {
```

Comment:
missing bound check advocated by the `@throws` clause

Reply:
Done